core/src/test/scala/integration/kafka/api/AbstractConsumerTest.scala

```diff
@@ -334,7 +334,7 @@ abstract class AbstractConsumerTest extends BaseRequestTest {
     @volatile var thrownException: Option[Throwable] = None
     @volatile var receivedMessages = 0
 
-    @volatile private var partitionAssignment: mutable.Set[TopicPartition] = new mutable.HashSet[TopicPartition]()
+    private val partitionAssignment: mutable.Set[TopicPartition] = new mutable.HashSet[TopicPartition]()
```

Comment:
nit: could we move this so that it's structured? e.g `vals` first, then `volatile vars`, then `vars` or something similar

Reply:
Great. Reordered.